.github/workflows/auto-release.yaml

```diff
@@ -0,0 +1,13 @@
+name: Auto Release CSM-Replication
+on:  # yamllint disable-line rule:truthy
+  workflow_dispatch:
+  repository_dispatch:
+    types: [auto-release-workflow]
+
+jobs:
+  csm-release:
+    uses: dell/common-github-actions/.github/workflows/csm-release-driver-module.yaml@main
+    with:
+      version: "minor"
+      image: "dell-csi-replicator,dell-replication-controller"
```

Comment:
This repository manages 4 images. migrator and node-rescanner images in addition to above 2 images.

Reply:
Added the remaining images